indico_zodbimport/modules/event_papers.py

```diff
@@ -304,7 +304,7 @@ def _migrate_revisions(self, contribution, rm):
                 review = self._migrate_review(contribution, old_layout_review, PaperReviewType.layout)
                 revision.reviews.append(review)
                 review_colors += _review_color(review, 'L')
-            contribution.paper_revisions.append(revision)
+            contribution._paper_revisions.append(revision)
```

Comment:
We thought about it, but since it's the zodb import and we only run it once we kept it simple.